services/core-api/app/api/verifiable_credentials/models/w3c_map_credentials.py

```diff
@@ -0,0 +1,28 @@
+
+from sqlalchemy.dialects.postgresql import UUID
+from sqlalchemy.schema import FetchedValue
+from app.extensions import db
+from typing import List
+from app.api.utils.models_mixins import AuditMixin, Base
+from app.api.mines.permits.permit.models.permit import Permit
+from app.api.mines.permits.permit_amendment.models.permit_amendment import PermitAmendment
+
+class UNTPCCredential():
+    pass
+
+
+class PartyVerifiableCredentialW3CCredential(AuditMixin, Base):
+    """Verificable Credential"""
+    __tablename__ = "party_verifiable_credential_w3c_credential"
```

Comment:
We'll need to add a Flyway migration to match this new table otherwise this will cause some issues when deployed to dev

Reply:
I'll remove this. it's currently unused, good catch.